docs/framework/whats-new/obsolete-members.md

```diff
@@ -496,13 +496,13 @@ The tables in this article list the type members that are obsolete in the .NET F
 |<xref:System.ComponentModel.MemberDescriptor?displayProperty=nameWithType>|<xref:System.ComponentModel.MemberDescriptor.GetInvokee%2A>|This method has been deprecated. Use <xref:System.ComponentModel.MemberDescriptor.GetInvocationTarget%2A?displayProperty=nameWithType> instead.|  
```

Comment:
lines 377, 378, 380, 381, 383, 384, 387, 388 last link should be System.Security.Permissions.PermissionState.None